src/helpers/figures/charts/pie_chart.ts

```diff
@@ -213,7 +213,21 @@ function getPieConfiguration(
   const fontColor = chartFontColor(chart.background);
   const config = getDefaultChartJsRuntime(chart, labels, fontColor, localeFormat);
   const legend: DeepPartial<LegendOptions<"pie">> = {
-    labels: { color: fontColor },
+    labels: {
+      color: fontColor,
+      usePointStyle: true,
+      //@ts-ignore
+      generateLabels: (_chart) =>
+        //@ts-ignore
+        _chart.data.labels.map((label, index) => ({
+          text: label,
+          strokeStyle: _chart.data.datasets[0].backgroundColor![index],
```

Comment:
datasets[0] ? Why does this one loop on the labels and not on the datasets ?

Reply:
Because the legend of a pie chart doesn't contains one entry per series (like in other chart) but one entry per element of a series.